Submitted By: Pierre Labastie <pierre dot labastie at neuf dot fr>
Date: 2020-05-20
Initial Package Version: 1.1.7
Upstream Status: Defunct
Origin: Self
Description: Fixes to allow building with -fno-common.

diff -Naur transcode-1.1.7.old/import/demux_pass.c transcode-1.1.7.new/import/demux_pass.c
--- transcode-1.1.7.old/import/demux_pass.c	2011-11-19 17:50:27.000000000 +0100
+++ transcode-1.1.7.new/import/demux_pass.c	2020-05-20 09:35:07.207493221 +0200
@@ -32,7 +32,7 @@
 #include "packets.h"
 
 
-int gop, gop_pts, gop_cnt;
+// int gop, gop_pts, gop_cnt;
 
 /* if you listen carefully, then you can hear the desesperate
  * whisper of this code calling for a rewrite. Or for a redesign.
diff -Naur transcode-1.1.7.old/import/extract_ac3.c transcode-1.1.7.new/import/extract_ac3.c
--- transcode-1.1.7.old/import/extract_ac3.c	2011-11-19 17:50:27.000000000 +0100
+++ transcode-1.1.7.new/import/extract_ac3.c	2020-05-20 09:35:31.697199286 +0200
@@ -334,7 +334,7 @@
 FILE *fd;
 
 #define MAX_BUF 4096
-char audio[MAX_BUF];
+static char audio[MAX_BUF];
 
 
 /* from ac3scan.c */
diff -Naur transcode-1.1.7.old/import/extract_mp3.c transcode-1.1.7.new/import/extract_mp3.c
--- transcode-1.1.7.old/import/extract_mp3.c	2011-11-19 17:50:27.000000000 +0100
+++ transcode-1.1.7.new/import/extract_mp3.c	2020-05-20 09:35:48.271000352 +0200
@@ -243,7 +243,7 @@
 }
 
 #define MAX_BUF 4096
-char audio[MAX_BUF];
+static char audio[MAX_BUF];
 
 /* ------------------------------------------------------------
  *
diff -Naur transcode-1.1.7.old/import/extract_pcm.c transcode-1.1.7.new/import/extract_pcm.c
--- transcode-1.1.7.old/import/extract_pcm.c	2011-11-19 17:50:27.000000000 +0100
+++ transcode-1.1.7.new/import/extract_pcm.c	2020-05-20 09:35:37.990123753 +0200
@@ -33,7 +33,7 @@
 #include "tc.h"
 
 #define MAX_BUF 4096
-char audio[MAX_BUF];
+static char audio[MAX_BUF];
 
 #define BUFFER_SIZE 262144
 static uint8_t buffer[BUFFER_SIZE];
